datafusion/sqllogictest/test_files/order.slt

```diff
@@ -1196,3 +1196,130 @@ physical_plan
 02)--RepartitionExec: partitioning=RoundRobinBatch(2), input_partitions=1
 03)----SortExec: TopK(fetch=1), expr=[a@0 ASC NULLS LAST], preserve_partitioning=[false]
 04)------CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_2.csv]]}, projection=[a, b], has_header=true
+
+
+####################
+# Test issue: TBD
+####################
```

Comment:
I just realized I put these tests in `order` (not `limit`).
I'll add a followup commit added now, to not mess up the commit history. Apologies.